apps/timetable/admin/ui/js/index.js

```diff
@@ -125,7 +125,7 @@ define(['gh.core', 'gh.constants', 'gh.admin-listview', 'gh.admin-batch-edit', '
                             'canManage': course.canManage,
                             'part': part,
                             'isEditing': false, // TODO: Replace when locking is supported in the backend
```

Comment:
This is not on you, but can you update `isEditing` as well here? Further down in the file we populate it with `part.Group.LockedBy || false`.